### ISSUE TITLE
Make the sandstorm update story clear for self-hosters

### ISSUE THREAD

**Issue by phildini** [sandstorm-io/sandstorm#362]
Right now, after looking at the README and wiki, I couldn't figure out as a self-hoster how to update/upgrade sandstorm itself. Some options present themselves.

1) This is actually already well-documented, but I missed it. Could be PEBKAC, but might indicate the docs need another look.
2) Sandstorm already self-updates, and doesn't need intervention. Awesome! Documentation explaining that would be really helpful.
3) There are commands that I as a self-hoster should run regularly. Documentation explaining this seems critical.

Thanks for all the great work you're doing!

**Comment by paulproteus** [comment]
Number 2 is accurate. But if you opt out of auto-update, then ???

We should doc this + put it in the https://github.com/sandstorm-io/sandstorm/wiki/Self-hosting-FAQ in the absence of a better idea.

**Comment by phildini** [comment]
Agreed! FWIW, I thought that (2) was what was going on, and even looked in the FAQ expecting to see a line about it, but no line was there. So you're super close to a great self-solving problem for self-hosters. :)

**Comment by paulproteus** [comment]
There is also

`$ sandstorm update`

which takes a command line parameter of a "bundle file", which is a tar.xz file e.g. https://dl.sandstorm.io/sandstorm-76.tar.xz

**Comment by ritjoe** [comment]
Just posting to note that the info previously in the github wiki link above has moved to 

https://docs.sandstorm.io/en/latest/administering/faq/

While the body of the github wiki page above notes this, it might save someone reading this issue a few moments and some frustration if they can go straight to it instead of having to make a stop along the way.
